go/tools/builders/protoc.go

```diff
@@ -57,14 +57,23 @@ func run(args []string) error {
 	if err := flags.Parse(args); err != nil {
 		return err
 	}
+
+	// Output to a temporary folder and then move the contents into place below.
+	// This is to work around long file paths on Windows.
+	tmpDir, err := ioutil.TempDir("", "go_proto")
+	if err != nil {
+		return err
+	}
+	tmpDir = abs(tmpDir)
+
```

Comment:
`defer os.RemoveAll(tmpDir)` instead of calling it at the end.

Reply:
Done.